book/src/getting-started.md

```diff
@@ -35,7 +35,7 @@ It is also possible to specify other users with the configuration file.
 
 ```bash
 cat <<EOF | adduser -w none -f -
-tests::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
+dummy::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
```

Comment:
After the user is done with pjdfstest, he's going to see "dummy" in the passwd file and not know where it came from.  Better to give the user the same name as the package.  That's the convention behind other packages that must create users.
```suggestion
pjdfstest::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
```